tutorials/tour/_posts/2017-02-13-basics.md

```diff
@@ -136,7 +141,7 @@ def add(x: Int, y: Int): Int = x + y
 println(add(1, 2)) // 3
 ```
 
-Methods cannot be named with the `val` or `var` keywords.
+Notice how the return type is declared _after_ the parameter list: `: Int`. Methods cannot be named with the `val` or `var` keywords.
```

Comment:
> Methods cannot be named with the `val` or `var` keywords.

not sure about this addition. is this really needed here? if so, I think you need to expand on this and be clearer. (else, omit?) if you want to include it, I would suggest stating it positively rather than negatively: "A method is always defined using the `def` keyword."